crates/matrix-sdk-indexeddb/src/crypto_store.rs

```diff
@@ -1212,6 +1274,42 @@ impl Drop for IndexeddbCryptoStore {
     }
 }
 
+/// The objects we store in the gossip_requests indexeddb object store
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+struct GossipRequestIndexedDbObject {
+    /// encrypted hash of the SecretInfo structure
+    info: String,
+
+    /// encrypted serialised representation of the GossipRequest as a whole
+    request: Vec<u8>,
+
+    /// whether the request has yet to be sent out
+    //
+    // Booleans don't really do what you might hope in indexeddb. Instead, we use an `Option`, and
+    // omit the value altogether if it is `None`, which means it will be excluded from the
+    // "unsent" index. If it is `Some`, the actual value is unimportant.
```

Comment:
```suggestion
    ///
    /// Booleans don't really do what you might hope in indexeddb. Instead, we use an `Option`, and
    /// omit the value altogether if it is `None`, which means it will be excluded from the
    /// "unsent" index. If it is `Some`, the actual value is unimportant.
```

Reply:
I believe that's really important :-p.